tidb-lightning/tidb-lightning-physical-import-mode.md

```diff
@@ -62,7 +62,7 @@ It is recommended that you allocate CPU more than 32 cores and memory greater th
 
 - Do not use the physical import mode to directly import data to TiDB clusters in production. It has severe performance implications. If you need to do so, refer to [Import data into a cluster in production](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#import-data-into-a-cluster-in-production).
 - Do not use multiple TiDB Lightning instances to import data to the same TiDB cluster by default. Use [Parallel Import](/tidb-lightning/tidb-lightning-distributed-import.md) instead.
-- When you use multiple TiDB Lightning to import data to the same target, do not mix the backends. That is, do not use the physical import mode and the logical import mode at the same time.
+- When you use multiple TiDB Lightning to import data to the same target table, do not mix the import modes. That is, do not use the physical import mode and the logical import mode at the same time.
```

Comment:
Should this refer to same target cluster instead of table? Basically, TiDB physical mode would change pd and tikv mode for fast importing data, and that would impact the use of tidb.

Reply:
Hey @ran-huang , would you like to check this comment?

Reply:
Sorry for the late response. 
@lance6716 Can you take a look at this? Is this the same "target table" or "target cluster"?

Reply:
yes this case is target cluster

Reply:
@db-will Fixed. PTAL